src/main/java/entity/Property.java

```diff
@@ -27,6 +27,7 @@ public Property(int ID, String name, ArrayList<Integer> priceList, ArrayList<Int
         this.tax = 0;
         this.priceList = priceList; // priceList sample: [15, 30, 45, 0]
         this.taxList = taxList; // taxList sample: [0, 10, 20, 30]
+        this.owner = null;
```

Comment:
Is that necessary? I thought they are automatically null 😨

Reply:
Oh, I wrote this a long time ago. Can be deleted, but you know, I prefer writing something like 'if (choice == true)' instead of if (choice) in the first place for clarification. I guess this '= null' initialization is the same

Reply:
> Is that necessary? I thought they are automatically null 😨

They are automatically `null`, but not explicitly writing it out may cause some problems, e.g.: 
When I was writing test for something else, I didn't know that this has a chance to be `null`, instead of having some `Player` class being there. 

I assumed there was a `Player` class there, which caused some issues.

Reply:
I see, I'm totally fine with that